scripts/operation/start-trubudget.sh

```diff
@@ -0,0 +1,214 @@
+            # Enable Services
+            sed -i 's/EMAIL_SERVICE=.*/EMAIL_SERVICE=ENABLED/g' ${SCRIPT_DIR}/.env
+            sed -i 's/MULTICHAIN_FEED=.*/MULTICHAIN_FEED=ENABLED/g' ${SCRIPT_DIR}/.env
+            sed -i 's/REACT_APP_EMAIL_SERVICE_ENABLED=.*/REACT_APP_EMAIL_SERVICE_ENABLED=true/g' ${SCRIPT_DIR}/.env
+            ENABLED_SERVICES="${ENABLED_SERVICES} emaildb"
+            echo "INFO: email-service enabled"
+            
+            elif [ "$word" = "excel-export-service" ]; then
+            sed -i 's/REACT_APP_EXPORT_SERVICE_ENABLED=.*/REACT_APP_EXPORT_SERVICE_ENABLED=true/g' ${SCRIPT_DIR}/.env
+            echo "INFO: excel-export-service enabled"
+            
+            elif [ "$word" = "storage-service" ]; then
+            sed -i 's/DOCUMENT_FEATURE_ENABLED=.*/DOCUMENT_FEATURE_ENABLED=true/g' ${SCRIPT_DIR}/.env
+            ENABLED_SERVICES="${ENABLED_SERVICES} minio"
+            echo "INFO: storage-service enabled"
+            
+        else
+            echo "${red}ERROR: Unknown service $word${colorReset}"
+            echo "${red}Only these services can be added with --enable-service: email-service, excel-export-service, storage-service${colorReset}"
+            echo "${red}Also, make sure the option --enable-service is used as last option!${colorReset}"
+            exit 1
+        fi
+    done
+fi
+
+if [ "$IS_FULL" = true ]; then
+    # Full version: enable all services
+    sed -i 's/EMAIL_SERVICE=.*/EMAIL_SERVICE=ENABLED/g' ${SCRIPT_DIR}/.env
+    sed -i 's/MULTICHAIN_FEED=.*/MULTICHAIN_FEED=ENABLED/g' ${SCRIPT_DIR}/.env
+    sed -i 's/REACT_APP_EMAIL_SERVICE_ENABLED=.*/REACT_APP_EMAIL_SERVICE_ENABLED=true/g' ${SCRIPT_DIR}/.env
+    sed -i 's/REACT_APP_EXPORT_SERVICE_ENABLED=.*/REACT_APP_EXPORT_SERVICE_ENABLED=true/g' ${SCRIPT_DIR}/.env
+    sed -i 's/DOCUMENT_FEATURE_ENABLED=.*/DOCUMENT_FEATURE_ENABLED=true/g' ${SCRIPT_DIR}/.env
+fi
+
+
+
+if [ "$IS_FULL" = true ]; then
+    if [ "$WITH_PROVISIONING" = false ]; then
+        echo "INFO: Setup full TruBudget environment without provisioning ..."
+        COMPOSE_SERVICES="master-node emaildb minio master-api email-service excel-export-service storage-service frontend"
+    else
+        echo "INFO: Setup full TruBudget environment with provisioning ..."
+        # Empty means all containers
+        COMPOSE_SERVICES=""
+    fi
+else
+    if [ "$WITH_PROVISIONING" = false ]; then
+        # Default Setup here
+        echo "INFO: Setup slim TruBudget environment without provisioning ..."
+        COMPOSE_SERVICES="master-node master-api frontend"
+    else
+        echo "INFO: Setup slim TruBudget environment with provisioning ..."
+        COMPOSE_SERVICES="master-node master-api provisioning frontend"
+    fi
+fi
+
+COMPOSE="docker-compose -f $SCRIPT_DIR/docker-compose.yml -p trubudget-operation --env-file $SCRIPT_DIR/.env"
+$COMPOSE down
+
+echo "INFO: Pull images from https://hub.docker.com/ ..."
+$COMPOSE pull
+
+#ToDo: Only use Images from Dockerhub
```

Comment:
is todo done?

Reply:
this todo will be done in the follow up issue #932